### DISCUSSION TITLE
Package naming

### DISCUSSION THREAD

**Discussion by feledori** [pmndrs/jotai#1688] [Ideas]
I follow this project now quite a while and I noticed the frequent package name changes/inconsistencies. From jotai to jotai/package to jotai-package. Is there a specific reasoning behind the naming? Why not follow other tools with jotai for the core and @jotai/package for associated packages.

**Comment by dai-shi** [comment]
Sorry for the confusion. Toward v2, We are moving to minimal core + many add-on packages style for ecosystem. The goal is to invite more package maintainers. Originally, we hoped to use `@jotai/foo`, but this namespace is already taken. So, we go for `jotai-foo`, but as you guess this causes naming conflicts. We will see. At least, the current state is somewhat consistent.

**Reply by feledori** [reply]
Nothing to apologise for. But thanks for letting me know. Have you tried getting in contact with the other person owning the namespace?

**Reply by dai-shi** [reply]
Yeah, we sent an email before, but no response.

**Reply by feledori** [reply]
Ahh man that sucks... Maybe npm can do something about it, but probably not. And what a coincidence that someone else used jotai. Wouldn't call jotai a standard name.

**Reply by dai-shi** [reply]
Seems like it's Finnish dialect or something. btw, valtio is also from Finnish.
